.gitpod.yml

```
@@ -2,6 +2,8 @@ image: nfcore/gitpod:latest
 tasks:
   - name: install current state of nf-core/tools
     command: python -m pip install --upgrade -r requirements-dev.txt -e .
```

I have never seen pip used to install requirements from multiple sources at once. It may work, but I think a sequential installation might easier to troubleshoot.

And I think I would skip the `--upgrade` flag to hopefully make the codespace more reproducible.

```suggestion
    command: |
      python -m pip install -e .
      python -m pip install -r requirements-dev.txt
```